src/main/java/com/strandls/resource/dao/ResourceDao.java

```diff
@@ -67,45 +68,33 @@ public List<Resource> findByObjectId(List<Long> objectIds) {
 		return result;
 	}
 
-	@SuppressWarnings("unchecked")
 	public List<Long> getResourceIds(List<String> contexts, List<String> types, List<Long> users, List<Long> ids) {
-		String qry = "SELECT R.id FROM Resource R WHERE 1=1";
-
-		if (contexts != null && !contexts.isEmpty() && !contexts.contains("all")) {
-			qry += " AND R.context IN (:contexts)";
-		}
-
-		if (types != null && !types.isEmpty() && !types.contains("all")) {
-			qry += " AND R.type IN (:types)";
-		}
+		String qry = "SELECT R.id FROM Resource R";
```

Comment:
need to get rid of the sonar issue in this function and find a better way to accomplish this filter handling.

Reply:
reverting back to commit 0cfa8b746c2dadacc1e2e00afb1a3ba60a737388 because change of method causes sonar bug.